php/blocks/controls/class-user.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * User control.
+ *
+ * @package   Block_Lab
+ * @copyright Copyright(c) 2018, Block Lab
+ * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+namespace Block_Lab\Blocks\Controls;
+
+/**
+ * Class User
+ */
+class User extends Control_Abstract {
+
+	/**
+	 * Control name.
+	 *
+	 * @var string
+	 */
+	public $name = 'user';
+
+	/**
+	 * User constructor.
+	 */
+	public function __construct() {
+		parent::__construct();
+		$this->label = __( 'User', 'block-lab' );
+	}
+
+	/**
+	 * Register settings.
+	 *
+	 * @return void
+	 */
+	public function register_settings() {
+		$this->settings[] = new Control_Setting(
+			array(
```

Comment:
It would also be good to have a `help` option for help text.
```
		$this->settings[] = new Control_Setting(
			array(
				'name'     => 'help',
				'label'    => __( 'Help Text', 'block-lab' ),
				'type'     => 'text',
				'default'  => '',
				'sanitize' => 'sanitize_text_field',
			)
		);
```

Reply:
Good suggestion, https://github.com/getblocklab/block-lab/pull/233/commits/d80465db25a7c78261801ab85f92e9675cf06b13#diff-7a9a8c2db176ed99ec78aa840392c3e0R39 adds your code above.